core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -206,15 +207,19 @@ angular.module('oppia').directive('audioTranslationBar', [
               $scope.checkingMicrophonePermission = false;
 
               $scope.elapsedTime = 0;
-              $scope.timerInterval = $interval(function() {
-                $scope.elapsedTime++;
-                // $scope.recordingTimeLimit is decremented to
-                // compensate for the audio recording timing inconsistency,
-                // so it allows the server to accept the recording.
-                if ($scope.elapsedTime === $scope.recordingTimeLimit - 1) {
-                  $scope.stopRecording();
-                }
-              }, 1000);
+              OppiaAngularRootComponent.ngZone.runOutsideAngular(() => {
+                $scope.timerInterval = $interval(function() {
+                  OppiaAngularRootComponent.ngZone.run(() => {
+                    $scope.elapsedTime++;
+                    // $scope.recordingTimeLimit is decremented to
+                    // compensate for the audio recording timing inconsistency,
+                    // so it allows the server to accept the recording.
+                    if ($scope.elapsedTime === $scope.recordingTimeLimit - 1) {
+                      $scope.stopRecording();
+                    }
+                  });
+                }, 1000);
```

Comment:
`fileUploadFeatures` suite get stuck on the translation tab, once exploration page is used with base content (migrated).

This due to protractor waiting for the $interval async task to finish. But $interval in periodic task and never finishes.
So, we were getting script timeout on the suite.

Here, using this fix, we explicitly tell ngZone to not to wait for this interval async task.